src/store/modules/product/actions.ts

```diff
@@ -58,19 +58,14 @@ const actions: ActionTree<ProductState, RootState> = {
   },
 
   async updateAppliedFilters({ commit, state, dispatch }, payload) {
-    const value = payload.value
-    const appliedFilters = JSON.parse(JSON.stringify((state.appliedFilters as any)[payload.type][payload.id]))
-    appliedFilters.list.includes(value) ? appliedFilters.list.splice(appliedFilters.list.indexOf(value), 1) : appliedFilters.list.push(value)
-    commit(types.PRODUCT_FILTER_UPDATED, {id: payload.id, type: payload.type, value: appliedFilters})
+    commit(types.PRODUCT_FILTER_UPDATED, { id: payload.id, type: payload.type, value: payload.value })
     dispatch('updateQuery')
   },
 
   async updateAppliedFilterOperator({ commit, state, dispatch }, payload) {
-    const appliedFilters = JSON.parse(JSON.stringify((state.appliedFilters as any)[payload.type][payload.id]))
-    appliedFilters.operator = payload.value;
-    commit(types.PRODUCT_FILTER_UPDATED, {id: payload.id, type: payload.type, value: appliedFilters})
```

Comment:
Why this change is needed?
This action should only allow operator change. With this change, this action is updating Applied Filters like the above action with an exception of querying on condition

Reply:
Updated sir